assets/js/modules/adsense/index.js

```diff
@@ -79,3 +82,18 @@ addFilter(
 	'googlesitekit.AdSenseDashboardZeroDataRefactored',
 	fillFilterWithComponent( DashboardZeroData )
 );
+
+domReady( () => {
+	Widgets.registerWidget(
+		'adsenseSummary',
+		{
+			component: DashboardSummaryWidget,
+			width: Widgets.WIDGET_WIDTHS.HALF,
+			priority: 1,
+			wrapWidget: false,
```

Comment:
See above - unless this widget needs to use `Widget` with custom props, it should use `wrapWidget: true` here.

Reply:
@felixarntz I have updated this however, I needed to pass false here and use the `Widget` component in order to pass the correct class name.